recipes/check_shapes/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "check_shapes" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: check_shapes
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/check_shapes-{{ version }}.tar.gz
+  sha256: b699fcb1e60bb17e2c97007e444b89eeeea2a9102ff11d61fb52454af5348403
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
```

Think we need [`poetry-core` to build]( https://github.com/GPflow/check_shapes/blob/dec8ec9f32983bb2833c5fe6fa5fbbddb57c288f/pyproject.toml#L56-L57 ) as well

```suggestion
    - pip
    - poetry-core >=1.0.0
```